app/models/reports/region_summary_schema.rb

```diff
@@ -302,6 +302,12 @@ def visited_without_bp_taken(with_ltfu: false)
       end
     end
 
+    memoize def diabetes_treatment_outcome_breakdown_counts(blood_sugar_risk_state)
```

Comment:
Is it kinda weird that this method is called `_counts` and the internal method is called `_by_counts`? I see that the three pieces involved here are:

1. The method that gets the counts
1. The method that calculates rates based on (1)
1. The method that caches (1) 

What naming convention do we follow for HTN stuff? Maybe:

1. The method that gets the counts - `diabetes_treatment_outcome_breakdown_counts`
1. The method that calculates rates based on (1) - `diabetes_treatment_outcome_breakdown_rates`
1. The method that caches (1) - `diabetes_treatment_outcome_breakdown`

WDYT?

Reply:
@harimohanraj89, I have made these changes 
- The method that gets the counts - `diabetes_treatment_outcome_breakdown_counts`
   - Private method that gets the counts of a region - `diabetes_treatment_outcome_breakdown_counts_of_region`
- The method that calculates rates based on (1) - `diabetes_treatment_outcome_breakdown_rates`
  - Private method that gets the counts of a region `diabetes_treatment_outcome_breakdown_rates_of_region`

Reply:
Very cool very cool :shipit: